src/pages/Blog/BlogDetails/index.tsx

```
@@ -1,194 +1,520 @@
 import { useNavigate } from 'react-router-dom'
+import { useState } from 'react'
+import { Avatar, Tag } from 'antd'
+import {
+  ClockCircleOutlined,
+  EyeOutlined,
+  HeartOutlined,
+  HeartFilled,
+  ShareAltOutlined,
+  ArrowLeftOutlined,
```

ESLint

[error] 'ArrowLeftOutlined' is defined but never used.

(@typescript-eslint/no-unused-vars)